srml/session/src/lib.rs

```diff
@@ -161,20 +161,42 @@ decl_module! {
 	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
 		fn deposit_event<T>() = default;
 
-		/// Sets the session key of a validator (function caller) to `key`.
+		/// Sets the session key of a validator (`origin`) to `key`.
```

Comment:
I think both the original and the change are incorrect. You don't have to be a validator to call this.
```suggestion
		/// Sets the session key of the function caller to `key`.
```